webui/react/src/pages/ExperimentDetails/TrialsComparisonModal.tsx

```diff
@@ -253,7 +250,7 @@ const TrialsComparisonTable: React.FC<TableProps> = ({
           <div className={[css.row, css.spanAll].join(' ')}>
             <div className={[css.cell, css.spanAll].join(' ')}>
               Metrics
-              <MetricSelectFilter
+              <MetricSelect
```

Comment:
The placeholder issue

https://user-images.githubusercontent.com/109113616/222299811-3aee27ed-47a0-48c6-a5ec-3e7952f21304.mov